fairytale.md

```diff
@@ -6,7 +6,8 @@ In a faraway future, in a distant land
 lives the *Squonk*
 
 The average Squonk is 7 meters tall and screams loud.
-
+the squank is yellow and covered in scales. It hisses uncontrollably. 
```

Comment:
> squank

Hi! Is this a typo or it is intended? Do you want to introduce a new character maybe? I'm fine with that :smiley: 
Otherwise correct spelling and capitalization!
```suggestion
the Squonk is yellow and covered in scales. It hisses uncontrollably. 
```